Gemfile.lock

```diff
@@ -1,13 +1,13 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.7.1)
+    activesupport (5.0.0.1)
```

Comment:
This got bumped to 5 when resolving a rebase conflict and I regenerated the Gemfile.lock. If this wasn't still in development I'd avoid this but as it's not.. It's not harmful.